DiscordBotApis/Data/GuildContext.cs

```diff
@@ -0,0 +1,17 @@
+﻿using DiscordBotApis.Models;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DiscordBotApis.Data
+{
+    public class GuildContext : DbContext
```

Comment:
There is usually a singular `DbContext` for the entire project, or at least per Database.  So this would be something like `DiscordBotApisDbContext`, not `GuildContext`.

Reply:
fixed